google_drive/tests/__init__.py

```diff
@@ -0,0 +1 @@
+""" Put tests here """
```

Comment:
Maybe a more descriptive docstring here -- my first thought was 'Um, ok, so where are the tests?"